Update tutorial to use autolocking (#12859)
ocaml/dune: doc/tutorials/dune-package-management/setup.md

After our project skeleton is set up, we can proceed to the next step.

## Enable package management

So far we have done everything as in a regular Dune project. However, now we
need to tell Dune that we want to use the package management feature. To do so
we have to add a new stanza to our `dune-workspace` file, creating it if it
doesn't exist.

::::{dropdown} `dune-workspace`
:icon: file-code

:::{literalinclude} setup/dune-workspace
:language: dune
:emphasize-lines: 2
:::

This new stanza will tell Dune to determine the packages that your project

> depends on, find the right versions, download and build them on the next build.

Reviewer: ```suggestion
This new stanza will tell Dune to determine the packages that your project
depends on, find the right versions, download, and build them on the next build.
```

dune build
```

Given this is the first time we run the build system after enabling package

> management a number of things will happen:

Reviewer: ```suggestion
Since this is the first time we have run the build system after enabling package
management a number of things will happen:
```

```

:::{note}

> If you don't want to build your project and stop at step 5, you can use

Reviewer: ```suggestion
If you don't want to build your project, instead stopping at step 5, you can use
```
